target/linux/ath79/dts/ar9331_zsun_sd100.dts

```diff
@@ -0,0 +1,149 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar9331.dtsi"
+
+/ {
+	model = "Zsun SD100";
+	compatible = "zsun,sd100", "qca,ar9331";
+
+	aliases {
+		led-boot = &status;
+		led-failsafe = &status;
+		led-running = &status;
+		led-upgrade = &status;
+		serial0 = &uart;
+		label-mac-device = &wmac;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		status: status {
```

Comment:
Please use "led_" prefix for the node label, i.e. "led_status: status {"
Change aliases accordingly.

Reply:
The prefix "led_" was added as requested.